examples/Microsoft.Spark.CSharp.Examples/MachineLearning/Sentiment/Program.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.Spark.Sql;
+
+namespace Microsoft.Spark.Examples.MachineLearning.Sentiment
+{
+    /// <summary>
+    /// Example of using ML.NET + .NET for Apache Spark
+    /// for sentiment analysis.
+    /// </summary>
+    //public class Program
+    internal sealed class Program : IExample
+    {
+        //public static void Main(string[] args)
+        public void Run(string[] args)
+        {
+            if (args.Length != 1)
+            {
+                Console.Error.WriteLine(
+                    "Usage: <path to Resources folder with yelp.csv and MLModel.zip>");
+                Environment.Exit(1);
+            }
+
+            SparkSession spark = SparkSession
+                .Builder()
+                .AppName(".NET for Apache Spark Sentiment Analysis")
+                .GetOrCreate();
+
+            // Read in and display Yelp reviews
+            DataFrame df = spark
+                .Read()
+                .Option("header", true)
+                .Option("inferSchema", true)
+                .Csv(args[0] + "yelp.csv");
+                //.Csv(@"Resources\yelp.csv");
+            df.Show();
+
+            // Use ML.NET in a UDF to evaluate each review 
+            spark.Udf().Register<string, bool>(
+                "MLudf",
+                (text) => Sentiment(text, args[0] + "MLModel.zip"));
+
+            // Use Spark SQL to call ML.NET UDF
+            // Display results of sentiment analysis on reviews
+            df.CreateOrReplaceTempView("Reviews");
+            DataFrame sqlDf = spark.Sql("SELECT Column1, MLudf(Column1) FROM Reviews");
+            sqlDf.Show();
+
+            // Print out first 20 rows of data
+            // Prevents data getting cut off (as it is when we print a DF)
+            IEnumerable<Row> rows = sqlDf.Take(20);
+            foreach (Row row in rows)
+            {
+                Console.WriteLine(row);
+            }
```

Comment:
`DataFrame.Show()` takes in parameters to adjust how many rows to show.

Reply:
I wanted to display each row completely (rather than getting cutoff as it does with a DataFrame), thus I chose to convert to an IEnumerable and then display each row object. Unless there's a way to `DataFrame.Show()` without cutting off any of the output?

Reply:
I believe you can set the `truncate` parameter to 0.

Reply:
Did not know about this parameter, changed now. Thanks!